crates/bevy_ui/src/focus.rs

```diff
@@ -98,7 +94,12 @@ pub fn ui_focus_system(
     }
 
     let mouse_clicked =
-        mouse_button_input.just_pressed(MouseButton::Left) || touches_input.just_pressed(0);
+        mouse_button_input.just_pressed(MouseButton::Left) || touches_input.any_just_pressed();
+
+    let cursor_position = windows
+        .get_primary()
+        .and_then(|window| window.cursor_position())
+        .or_else(|| touches_input.first_pressed_position());
```

Comment:
Does this fix whatever "touch-position doesn't update"-bug that #4352 is trying to fix? I'm a bit unclear what that bug actually is, and this has `something` related to touch position so I'd figured I'd ask.

Reply:
I don't know, #4352 seems to attempt to fix internal touch position update, so I guess not?
What I'm doing is considering basically any touch position as a cursor position